irohad/main/application.cpp

```diff
@@ -239,7 +238,7 @@ void Application::initQueryService() {
   log_->info("[Init] => query service");
 }
 
-void Application::run(const Torii &torii) {
+void Application::run(const iroha::torii::config::Torii &torii) {
```

Comment:
Maybe remove `using namespace` decl then?

Reply:
It would be good probably to just write using.